etl-destinations/src/schema.rs

```diff
@@ -0,0 +1,82 @@
+use etl::types::Type;
+
+/// Returns whether the Postgres type is an array type.
+pub fn is_array_type(typ: &Type) -> bool {
```

Comment:
I would move this `etl-postgres`.

Reply:
Moved to `types` module in `etl-postgres`.